test/deploy_and_run_clickhouse_macos.sh

```diff
@@ -17,12 +17,13 @@ echo 1. Prepare folders
 mkdir -p usr/bin etc/clickhouse-server var/lib/clickhouse
 
 echo 2. Download binaries
-curl https://clickhouse-builds.s3.yandex.net/0/381947509a4f66236f943beaefb0b1f5c2fd979d/1570028580_binary/clickhouse -o usr/bin/clickhouse
-curl https://clickhouse-builds.s3.yandex.net/0/381947509a4f66236f943beaefb0b1f5c2fd979d/1570028580_binary/clickhouse-odbc-bridge -o usr/bin/clickhouse-odbc-bridge
+# TODO: switch to actual 20.3 binaries once available. Currently, this is master branch.
+curl https://clickhouse-builds.s3.yandex.net/0/d147cd646a1e6cc41d42f92d57f016a5c49d04de/clang-10-darwin_relwithdebuginfo_none_bundled_unsplitted_disable_False_binary/clickhouse -o usr/bin/clickhouse
+curl https://clickhouse-builds.s3.yandex.net/0/d147cd646a1e6cc41d42f92d57f016a5c49d04de/clang-10-darwin_relwithdebuginfo_none_bundled_unsplitted_disable_False_binary/clickhouse-odbc-bridge -o usr/bin/clickhouse-odbc-bridge
```

Comment:
Just after our talk yesterday Yandex guys added that: 
https://github.com/ClickHouse/ClickHouse/pull/11889/files

But let's leave it as is for now.

Reply:
I checked the link, they don't have `.../20.3/...` binary there, looks like.